workbench-core/example/infrastructure/README.md

```diff
@@ -14,31 +14,55 @@
  5. Install [jq](https://stedolan.github.io/jq/) command line tool (required for `./scripts/getResources.sh` script) by running e.g. `brew install jq`
  6. Install [AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
 
- ## Setup your environment for Integration Test
- Navigate to `workbench-core/example/infrastructure`
+## Setup your main environment for Integration Test
+Navigate to `workbench-core/example/infrastructure`
+
+### Set Environment Variables
+```bash
+export HOSTING_ACCOUNT_ID=<HOSTING_ACCOUNT_ID>
+export HOSTING_ACCOUNT_REGION=<HOSTING_ACCOUNT_REGION>
+export MAIN_ACCOUNT_ID=<MAIN_ACCCOUNT_ID>
+export MAIN_ACCOUNT_REGION=<MAIN_ACCCOUNT_REGION>
```

Comment:
would it make sense to include these into a config file instead?

Reply:
We cannot push configs with AccountIds, these instructions are for your local testing. In GitHub workflow these variables are read from the secrets